src/ROUTES.ts

```diff
@@ -78,6 +78,10 @@ export default {
     SETTINGS_ADD_DEBIT_CARD: 'settings/wallet/add-debit-card',
     SETTINGS_ADD_BANK_ACCOUNT: 'settings/wallet/add-bank-account',
     SETTINGS_ENABLE_PAYMENTS: 'settings/wallet/enable-payments',
+    SETTINGS_WALLET_CARDS_DIGITAL_DETAILS_UPDATE_ADDRESS: {
+        route: 'settings/wallet/cards/:domain/digital-details/update-address',
```

Comment:
Actually I think it should be `card` and not `cards`. Maybe it would help to use the prefix here for consistency:

https://github.com/Expensify/App/blob/cb883e89e54ec3304fdf7f5c0e0f10fee3cc4ab9/src/ROUTES.ts#L74-L75

Reply:
Good catch Marc, I agree. @MrMuzyk

Reply:
Fixing that! There was a mistake in docs then
<img width="528" alt="image" src="https://github.com/Expensify/App/assets/23487612/9475f9df-50eb-4567-8309-2d48f98abd9d">

Reply:
looking at the codebase we actually have more cases of /cards than /card ATM. Should I change all of the to `/card`?

<img width="864" alt="image" src="https://github.com/Expensify/App/assets/23487612/293885d2-219e-48f1-a7bc-a478dbb8ffa5">

Reply:
I'd go head and update them. Would be good to be consistent about this one.